diagnostic.md

```diff
@@ -62,7 +62,7 @@ var result = strangeAdd(0, -5);
 
 What is the value of result?  Explain your answer.
 
-_Replace this text with your answer._
+The value of the result is `-5`. The first if expression says if a > 0 OR b > 0, then to return a + b. Since a = 0, only b > 0 satisfies and thus a + b is returned, meaning 0 + -5 is returned, which = -5. Since a return basically tells an expression to end immediately, the second return is not hit and not needed so the expression evaluates to -5.
```

Comment:
Value of result is -5. This is because neither of the conditions are met to be in the `if` loop. Therefore moving to `a - b` as the expression. So since `a` argument is 0 and `b` argument is -5; 
` 0 - (-5) = 5`